pypesto/ensemble/ensemble.py

```diff
@@ -1253,3 +1269,65 @@ def calculate_cutoff(
 
     range = chi2.ppf(q=percentile / 100, df=df)
     return fval_opt + range
+
+
+def calculate_hpd(
+    result: Result,
+    burn_in: int = 0,
+    ci_level: float = .95,
+):
+    """
+    Calculate Highest Posterior Density (HPD) samples of pypesto sampling result.
+
+    The HPD is calculated for a user-defined credibility level (alpha). The HPD includes all
```

Comment:
Rephrase to match argument names? "alpha" is potentially confusing since the function doesn't take an alpha parameter.